ddtrace/llmobs/_integrations/crewai.py

```diff
@@ -224,7 +225,7 @@ def _llmobs_set_span_link_on_task(self, span, args, kwargs):
         queued_task_node = self._crews_to_tasks.get(crew_id, {}).setdefault(str(queued_task_id), {})
         span_links = []
 
-        if queued_task.context:
+        if hasattr(queued_task, "context") and isinstance(queued_task.context, Iterable):
```

Comment:
Can we do a simpler check like this?
```suggestion
        if isinstance(getattr(queued_task, "context", None), Iterable)):
```